{"uid":"54693bd547511a09","name":"test_split_format_sql[select * from t where id == ?-params3-result3]","fullName":"tests.unit.async_db.test_typing_format#test_split_format_sql","historyId":"2ece2a108341cb84dbd1cec058545e1f","time":{"start":1682272301524,"stop":1682272301525,"duration":1},"status":"broken","statusMessage":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","statusTrace":"self = <allure_pytest.listener.AllureListener object at 0x7f33b2398710>\nfixturedef = <FixtureDef argname='result' scope='function' baseid=''>\n\n    @pytest.hookimpl(hookwrapper=True)\n    def pytest_fixture_post_finalizer(self, fixturedef):\n        yield\n        if hasattr(fixturedef, 'cached_result') and self._cache.get(fixturedef):\n            container_uuid = self._cache.pop(fixturedef)\n>           self.allure_logger.stop_group(container_uuid, stop=now())\n\n/opt/hostedtoolcache/Python/3.7.16/x64/lib/python3.7/site-packages/allure_pytest/listener.py:174: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.7.16/x64/lib/python3.7/site-packages/allure_commons/reporter.py:91: in stop_group\n    plugin_manager.hook.report_container(container=group)\n/opt/hostedtoolcache/Python/3.7.16/x64/lib/python3.7/site-packages/allure_commons/logger.py:39: in report_container\n    self._report_item(container)\n/opt/hostedtoolcache/Python/3.7.16/x64/lib/python3.7/site-packages/allure_commons/logger.py:30: in _report_item\n    with io.open(self._report_dir / filename, 'w', encoding='utf8') as json_file:\n/opt/hostedtoolcache/Python/3.7.16/x64/lib/python3.7/site-packages/pyfakefs/fake_io.py:117: in open\n    file, mode, buffering, encoding, errors, newline, closefd, opener\n/opt/hostedtoolcache/Python/3.7.16/x64/lib/python3.7/site-packages/pyfakefs/fake_open.py:97: in __call__\n    return self.call(*args, **kwargs)\n/opt/hostedtoolcache/Python/3.7.16/x64/lib/python3.7/site-packages/pyfakefs/fake_open.py:200: in call\n    file_object, file_path, open_modes, real_path\n/opt/hostedtoolcache/Python/3.7.16/x64/lib/python3.7/site-packages/pyfakefs/fake_open.py:299: in _init_file_object\n    real_path, create_missing_dirs=False, apply_umask=True\n/opt/hostedtoolcache/Python/3.7.16/x64/lib/python3.7/site-packages/pyfakefs/fake_filesystem.py:2359: in create_file_internally\n    self.raise_os_error(errno.ENOENT, parent_directory)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <pyfakefs.fake_filesystem.FakeFilesystem object at 0x7f33b1ebddd0>\nerr_no = 2\nfilename = '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'\nwinerror = None\n\n    def raise_os_error(\n        self,\n        err_no: int,\n        filename: Optional[AnyString] = None,\n        winerror: Optional[int] = None,\n    ) -> NoReturn:\n        \"\"\"Raises OSError.\n        The error message is constructed from the given error code and shall\n        start with the error string issued in the real system.\n        Note: this is not true under Windows if winerror is given - in this\n        case a localized message specific to winerror will be shown in the\n        real file system.\n    \n        Args:\n            err_no: A numeric error code from the C variable errno.\n            filename: The name of the affected file, if any.\n            winerror: Windows only - the specific Windows error code.\n        \"\"\"\n        message = os.strerror(err_no) + \" in the fake filesystem\"\n        if winerror is not None and sys.platform == \"win32\" and self.is_windows_fs:\n            raise OSError(err_no, message, filename, winerror)\n>       raise OSError(err_no, message, filename)\nE       FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'\n\n/opt/hostedtoolcache/Python/3.7.16/x64/lib/python3.7/site-packages/pyfakefs/fake_filesystem.py:418: FileNotFoundError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"global_fake_fs","time":{"start":1682272301519,"stop":1682272301522,"duration":3},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"afterStages":[{"name":"global_fake_fs::0","time":{"start":1682272301528,"stop":1682272301529,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"labels":[{"name":"parentSuite","value":"tests.unit.async_db"},{"name":"suite","value":"test_typing_format"},{"name":"host","value":"fv-az306-45"},{"name":"thread","value":"2398-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.unit.async_db.test_typing_format"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"params","value":"((1,),)"},{"name":"query","value":"'select * from t where id == ?'"},{"name":"result","value":"['select * from t where id == 1']"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"54693bd547511a09.json","parameterValues":["((1,),)","'select * from t where id == ?'","['select * from t where id == 1']"]}